### PR TITLE
ADD: wetterdienst

### PR DESCRIPTION
**Pull request by kmuehlbauer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
I'm confirming that I'm willing to be listed as maintainer.

**Comment by kmuehlbauer** [comment]
@conda-forge/staged-recipes This is ready for review/merge. Thanks!

File: recipes/wetterdienst/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c6be18b0cd82f634d385e8168193a8a6f42d9a24e66ac171fe33e07180aa3031
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```
Comment (ocefpaf) [inline recipes/wetterdienst/meta.yaml:15]: Use grayskull for creating the recipe and you'll get the latest "script" line here.

Comment (kmuehlbauer) [reply recipes/wetterdienst/meta.yaml:15]: @ocefpaf Thanks! Will try this next time.

**Pull request merged**